### ISSUE TITLE
web analytics without privacy concern

### ISSUE THREAD

**Issue by xhdong-umd** [ctmm-initiative/ctmmweb#127]
We discussed the issue about web analytics before. 
- The advantage is that we can have some ideas on how many users used our hosted app, and get some statistics on user's distribution.
- The disadvantage is that we don't want users to have concern about privacy or feeling they were being tracked. The most common tool google analytics doesn't pass this test.

Recently I noticed that there were some open source, privacy respected web analytics tool. One service claimed these points:

https://www.goatcounter.com/

- free for non-commercial use. We can sign up the service, insert a line of javascript in our app and that they would collect the data and keep them for 6 months.
- Privacy-aware; doesn’t track users with unique identifiers and doesn't need a GDPR notice.
- Identify unique visits without cookies using a non-identifiable hash.
- Keeps useful statistics such as browser information, location, and screen size. Keep track of referring sites and campaigns.

The list looks quite promising to me. @jmcalabrese , @chfleming , @NoonanM Do you guys think this worth a try? Are you comfortable with this kind of web analytics?

**Comment by jmcalabrese** [comment]
I agree this sounds promising. I suggest we give it a try and test it internally first to ensure that it both works and respects privacy as advertised. If it passes that test, then we can consider deploying it.

**Comment by xhdong-umd** [comment]
I tried with another shinyapps.io account to test it, but cannot make it work. I made a simple html page to host in github then tested again, still not working. I checked some other options, tried 2 more and still not working. 

Not sure what extra steps are needed. It was supposed to be just include some javascript code. Some other services require to install software in server, but we cannot do that so the options are limited.
